nrfu_tests/test_security_rp_username_password.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2023 Arista Networks, Inc.  All rights reserved.
+# Arista Networks, Inc. Confidential and Proprietary.
+
+"""
+Test cases for verification of username password status.
+"""
+
+import pytest
+from pyeapi.eapilib import EapiError
+from vane.logger import logger
+from vane.config import dut_objs, test_defs
+from vane import tests_tools
+
+TEST_SUITE = "nrfu_tests"
+
+
+@pytest.mark.nrfu_test
+@pytest.mark.security
+class SecurityRPUsernamePasswordTests:
```

Comment:
```suggestion
class SecurityRPUsernameNoPasswordTests:
```
can we update like this ?

Reply:
class name is updated as per discussion with vishwajeet